lib/DDG/Spice/Github.pm

```diff
@@ -19,7 +19,7 @@ spice proxy_cache_valid => '200 1d';
 
 handle query_lc => sub {
     s/^github\s*|\s+github$//;
-    if ($_ eq "" || m/\bjobs\b|\bstatus\b/) {
+    if ($_ eq "" || m/\bjobs\b|\bstatus\b|\bissue\b/) {
```

Comment:
@ManrajGrover please put these changes in another PR. We need to keep PRs to 1 IA each 👍